egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -111,6 +113,8 @@ def train_new_models(dir, iter, srand, num_jobs,
                                     if chunk_level_training
                                     else "--frame={0}".format(frame)),
                         momentum=momentum, max_param_change=max_param_change,
+                        scale_learning_rate=scale_learning_rate,
+                        scale_update_frequency=scale_update_frequency,
```

Comment:
I don't think it's necessary to have this frequency passed through the python scripts, we can just set a reasonable value at the code level.  but this is OK for now; it might not even work.